src/components/Proposals/ProposalBanner.js

```diff
@@ -73,19 +72,40 @@ function ProposalBanner({ onCloseBanner, ...props }) {
             color: ${theme.contentSecondary};
           `}
         >
-          We believe humankind should use technology as a liberating tool to
-          unleash all the goodwill and creativity of our species, rather than as
-          a tool to enslave and take advantage of one another.
+          This is a test <br /> Aragon Network Cash is a test organization set
+          up as part of a new round of precedence campaign disputes. Learn more
+          on the Aragon project blog
```

Comment:
```suggestion
          up as part of a new round of precedence campaign disputes. Learn more
          on the Aragon project blog.
```

I'm also wondering should we link to the post in this same sentence? e.g:

Learn more on the [Aragon project blog](https://aragon.org/blog).